include/klee/Support/ModuleUtil.h

```diff
@@ -39,6 +39,59 @@ std::unique_ptr<llvm::Module>
 linkModules(std::vector<std::unique_ptr<llvm::Module>> &modules,
             llvm::StringRef entryFunction, std::string &errorMsg);
 
+#if defined(__x86_64__) || defined(__i386__)
+#define addFunctionReplacement(from, to) \
+        {#from"f", #to"f"},              \
+        {#from, #to},                    \
+        {#from"l", #to"l"}               \
+
+#define addIntrinsicReplacement(from, to) \
+        {"llvm."#from".f32", #to"f"},     \
+        {"llvm."#from".f64", #to},        \
+        {"llvm."#from".f80", #to"l"}      \
+
```

Comment:
What about the intrinsics ending with ".f128"? For example, call of `__builtin_copysignf128` in C/C++ code generates call of `@llvm.copysign.f128` in IR code